helm/gorush/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Chart.Name }}-test-connection"
+  labels:
+    {{- include "gorush.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "gorush.fullname" . }}:{{ .Values.service.port }}']
+  restartPolicy: Never
```

Comment:
This should be handled by the health and ready checks on the pods. If you wanted an example consumer that should be a separate chart
```suggestion
```